packages/api/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@voltz-protocol/api-v2",
   "version": "1.35.0",
-  "main": "index.js",
+  "main": "dist/index.js",
```

Comment:
Why did we need to do this?

Reply:
removed "main" config [e0c8edf](https://github.com/Voltz-Protocol/v2-off-chain-monorepo/pull/34/commits/e0c8edf872ac0156572fc067bdb129d70fc36e1d)